### PR TITLE
doc: Update references to scenario options doc

### PR DESCRIPTION
**Pull request by amyreye**
Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

File: doc/tutorials/rtvm_performance_tips.rst [hunk 1]
```diff
@@ -189,8 +189,9 @@ Tip: Disable the software workaround for Machine Check Error on Page Size Change
    By default, the software workaround for Machine Check Error on Page Size
    Change is conditionally applied to the models that may be affected by the
    issue. However, the software workaround has a negative impact on
-   performance. If all guest OS kernels are trusted, the
-   :option:`hv.FEATURES.MCE_ON_PSC_DISABLED` option could be set for performance.
+   performance. If all guest OS kernels are trusted, you can disable the
```
Comment (dbkinder) [inline doc/tutorials/rtvm_performance_tips.rst:192]: Looks like the default changed to NOT enable this work around, so we need to change the text on the lines above and here.  @NanlinXie can you confirm?

**Review by dbkinder — CHANGES_REQUESTED** [review]
I think the default for 'Enable MCE workaround" changed

File: doc/tutorials/rtvm_performance_tips.rst (hunk 1 above)
Comment (NanlinXie) [reply doc/tutorials/rtvm_performance_tips.rst:192]: Yes , confirmed. Here we need uncheck "Enable MCE workaround" this option.

Comment (amyreye) [reply doc/tutorials/rtvm_performance_tips.rst:192]: @NanlinXie Not sure I understand your answer. Is the following text accurate?

> Tip: Disable the software workaround for Machine Check Error on Page Size Change.
> By default, the software workaround for Machine Check Error on Page Size Change is conditionally applied to the models that may be affected by the issue. However, the software workaround has a negative impact on performance. If all guest OS kernels are trusted, you can disable the software workaround (by changing the [:term:`Enable MCE workaround`](https://github.com/projectacrn/acrn-hypervisor/blob/17e7d00eb67cdbc38b665cb9d30104777e75d5a1/doc/tutorials/rtvm_performance_tips.rst#id3) option in the ACRN Configurator tool) for performance.

Comment (NanlinXie) [reply doc/tutorials/rtvm_performance_tips.rst:192]: The last sentence could be modified to "the "Enable MCE workaround" option could be unset for performance."

**Pull request merged**